Formula/g/ghostscript.rb

```diff
@@ -71,7 +70,7 @@ class Ghostscript < Formula
 
   def install
     # Delete local vendored sources so build uses system dependencies
-    libs = %w[expat freetype jbig2dec jpeg lcms2mt libpng openjpeg tiff zlib]
+    libs = %w[freetype jbig2dec jpeg lcms2mt libpng openjpeg tiff zlib]
```

Comment:
```suggestion
    libs = %w[expat freetype jbig2dec jpeg lcms2mt libpng openjpeg tiff zlib]
```
Let's not do this.

Reply:
yeah, the artifact was wrong, that is why, reverting this line.